_guide/_pages/interviews/wrap-up.md

```diff
@@ -56,26 +56,37 @@ Begin by introducing yourself by saying this or something similar to it:
 
-- ["Tell me about your experience with agile or other collaborative working styles. How does working in an agile way affect the technical choices you make?"](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.p3zgydisjuv3)
+- ["Whether or not you’ve had direct experience with agile, in an agile environment, how do you approach trade-offs?"](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.d4ctqn683zxs)
 
-- ["Have you had a chance to participate in user-centered design and/or user research?"](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.z49dyqcuxle5)
+- ["Tell me about your experience working on teams that directly involved non-engineers. How did you approach working with others with different professional skills?"](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.p2cr6nh5culs)
 
-- ["Have you ever worked with a remote/distributed team before?"](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.95oxlva5ufr7)
+- ["What has your experience been with user-centered design and/or user research?"](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.1407do81qrqo)
 
-- ["Please explain a complex technical concept or tool to me as if I were a non-technical partner"](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.8lizmty3jisl)
+### Technology and open source
+Ask one or more of the following:
+- ["Are you comfortable with sharing your work in a public fashion?"](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.hmo4ig69x3qn)
+- ["What is your comfort level with peer-reviewed work?"](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.hmo4ig69x3qn)
+- ["Your product owner is very excited about a proprietary subscription based product which would cost twice what an open source solution would. How would you encourage the product owner to use open source?"](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.hmo4ig69x3qn)
 
-- "At 18F we strive to be as representative of the populace as possible and work hard to ensure that everyone understands this notion and is treated fairly and respectfully. [What do diversity, equity, and inclusion mean to you?](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.5kwvj8b9hvhd)"
+### Communication and collaboration
+- ["How have you overcome any communication challenges and barriers in a remote/distributed environment? What crucial skills are needed to make a remote first team collaborative?"](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.3eohpexz765y)
```

Comment:
I might adjust the language here to be more about the candidate’s experience, and less about "book smarts": 

> How have you overcome any communication challenges and barriers in a remote/distributed environment? What tools or skills have you used to make a remote first team collaborative?

Reply:
+1